### ISSUE TITLE
.lproj/.strings resources copied multiple times

### ISSUE THREAD

**Issue by pietrocaselani** [tuist/tuist#361]
**Describe the bug**
I am configuring Tuist on my project, and I am facing a problem. I have a target that bundles strings files, and those files are been copied twice (for `en.lproj` and `pt.lproj`) and because of that there is this warning, twice:

<img width="936" alt="Screen Shot 2019-05-19 at 01 18 07" src="https://user-images.githubusercontent.com/862516/57977714-b8aec580-79d4-11e9-8217-3f3ea0026bd3.png">

```
Skipping duplicate build file in Copy Bundle Resources build phase: ~/CouchTracker/CouchTrackerCore/Resources/en.lproj/CouchTrackerCore.strings (in target 'CouchTrackerCore-iOS')
```
I tried the example #266 but didn't work.

Here is my [Project.swift file](https://github.com/pietrocaselani/CouchTracker/blob/tuist/Project.swift)
and [here is the line](https://github.com/pietrocaselani/CouchTracker/blob/tuist/Project.swift#L262) on the Project file that handles the framework resources.

And so far, I never managed to get one only file been copied and have both languages available.

So I don't know if I am doing something wrong, or if it's a bug.

**To Reproduce**
* You can clone the [project](https://github.com/pietrocaselani/CouchTracker/tree/tuist)
* Checkout to branch `tuist`
* run `setup.sh` on the root of the project
* run `tuist generate` on the root of the project

**Expected behavior**
I expect to see only one resource been copied, as was before creating the project using tuist

**Desktop (please complete the following information):**

- OS: `macOS 10.14.4 (18E226)`
- Xcode: `10.2.1 (10E1001)`
- Swift: `5.0.1`
- Tuist version: `0.14.0`

Let me know if you need any helping to "understand" the `Project.swift` file... it's big and I still don't the best way to organize everything.

**Comment by kwridan** [comment]
Thanks for flagging this @pietrocaselani, looks like a bug in the way localised files are added to the resources build phase.

Additionally duplicates appear to get added as file elements in the project navigator in the event multiple `.strings` files exist: 

<img width="197" alt="Screenshot 2019-05-19 at 10 15 40" src="https://user-images.githubusercontent.com/11914919/57980153-123be200-7a1f-11e9-9eec-c2b1d2ff36ff.png">

**Comment by kwridan** [comment]
Side note: thanks for sharing the example manifest file! It's always great to see how manifest files are used for various projects and use cases. 

Have you experimented with creating separate projects to group some of the targets? (i.e. having a workspace with multiple projects)

- CouchTracker Project _(possibly can also be split to CouchTracker and CouchTrackerCore)_
  - CouchTracker
  - CouchTrackerApp
  - ...
- Trakt Project
  - TraktSwift
  - ...
- TVDB Project
  - TVDBSwift
  - ...
- TMDB Project
  - TMDBSwift
  - ...

Traditionally this would have been a pain to manage with Xcode manually, but with tooling like Tuist it can make such a structure more appealing.

**Comment by pietrocaselani** [comment]
Glad to help @kwridan 

> "Have you experimented with creating separate projects to group some of the targets? (i.e. having a workspace with multiple projects)"

I tried this approach on the project at the company I am working.
We had multiple repositories, each one was a Xcode project or workspace (when using cocoapods).
Then we decided to group all repositories together, grouping all projects into a single workspace.

Worked really well, but I faced some issues, like sharing configuration across all the projects.
So for my personal project I decided to try one project and many targets.

But sure, now that I using tuist, to move to one workspace and many projects will be easier, and maybe I would give a try.

Let me know if I can help in anyway.
